recipes-security/optee/files/0001-mmu-ignore-VA-spaces-in-core_mmu_get_type_by_pa.patch

```diff
@@ -0,0 +1,39 @@
+From d0311fc02fec094f0f790e18a5d864bbb57931e4 Mon Sep 17 00:00:00 2001
+From: Volodymyr Babchuk <volodymyr_babchuk@epam.com>
+Date: Fri, 26 Jul 2024 16:15:17 +0300
+Subject: [PATCH 1/2] mmu: ignore VA spaces in core_mmu_get_type_by_pa
```

Comment:
I see PATCH 1/2 here. Maybe the second patch missing?

Reply:
Second patch is actually the first one, that adds RPI5 support. 

I ran `git format-patch -2` to get this one, as it precedes the patch with RPI5 support in the OP-TEE PR: https://github.com/OP-TEE/optee_os/pull/6854 

I didn't wanted to update both patches, so this is how it is.